Content.Shared/Chat/ChatModifiers/SanitizeRadioPrefixChatModifier.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Linq;
+using Content.Shared.CCVar;
+using Content.Shared.Chat.Prototypes;
+using Content.Shared.Decals;
+using Content.Shared.Radio;
+using Robust.Shared.Configuration;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Utility;
+
+namespace Content.Shared.Chat.ChatModifiers;
+
+/// <summary>
+/// Removes any radio chat prefixes from the first text node.
+/// </summary>
+[Serializable]
+[DataDefinition]
+public sealed partial class SanitizeRadioPrefixChatModifier : ChatModifier
+{
+
+    public override FormattedMessage ProcessChatModifier(FormattedMessage message, Dictionary<Enum, object> channelParameters)
+    {
+
+        var returnMessage = new FormattedMessage();
+        if (!message.Nodes.TryFirstOrDefault(x => x.Name == null, out var firstTextNode))
+        {
+            return message;
+        }
```

Comment:
new line after if block. maybe make it without braces? :< also move creation of FormattedMessage lower so u create only if u use